### PR TITLE
Enabled multiple selection and deletion in Asset Browser

### PR DESCRIPTION
**Pull request by utkarshgautamnyu**
With reference to https://highfidelity.fogbugz.com/f/cases/6513/Allow-a-user-to-select-more-than-one-thing-in-the-asset-browser
Allowed user to select multiple items in the asset browser (using ctrl+click or cmd+click and shift+click)
Multiple items can be deleted together

Testing Notes:
1) Run the built PR
2) Open Asset Browser
3) Click one item and ctrl+click (cmd+click) on another item -> Both the items should get selected
4) Click one item and shift+click on another item -> The entire range of items gets selected.
5) When multiple items are selected, observe "Add to World" and "Rename" buttons are disabled
6) Select multiple items and click "Delete"
7) Observe resulting modal displays the count of items marked for deletion.
8) Observe the selected items are deleted

Note: When a single item is selected, the Delete modal displays if the item being deleted is a file/folder and its path. When multiple items are selected, it just displays the number of items marked for deletion.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2055/HighFidelity-Beta-PR11064-07fc9a6dc322f56237394707c9770615a497debe.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2055/HighFidelity-Beta-PR11064-07fc9a6dc322f56237394707c9770615a497debe.exe

File: interface/resources/qml/AssetServer.qml [hunk 1]
```diff
@@ -289,23 +303,37 @@ ScrollingWindow {
         });
     }
     function deleteFile(index) {
+        var path=[];
```
Comment (birarda) [inline interface/resources/qml/AssetServer.qml]: spacing around `=` here

File: interface/resources/qml/hifi/dialogs/TabletAssetServer.qml [hunk 2]
```diff
@@ -290,23 +303,37 @@ Rectangle {
         });
     }
     function deleteFile(index) {
+        var path=[];
```
Comment (birarda) [inline interface/resources/qml/hifi/dialogs/TabletAssetServer.qml]: same comment here

**Comment by birarda** [comment]
Couple of spacing comments.

This isn't on you @utkarshgautamnyu but I think it's pretty sad that we have both AssetServer.qml and TabletAssetServer.qml when they are VERY similar. We should make a note to see if we can combine the two of them and simplify this code.

**Comment by utkarshgautamnyu** [comment]
@birarda I have made the changes. Agree on both AssetServer.qml and TabletAssetServer.qml being very similar, I had to duplicate the same code in both the files.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2061/HighFidelity-Beta-PR11064-1beec26003f47bdcedc1c0d2172b7ffe228d9ad4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2061/HighFidelity-Beta-PR11064-1beec26003f47bdcedc1c0d2172b7ffe228d9ad4.exe

**Comment by Nex-Pro** [comment]
Testing

**Comment by Nex-Pro** [comment]
### PASS

Test criteria met.

Run the built PR **DONE**
Open Asset Browser **DONE**
Click one item and ctrl+click (cmd+click) on another item -> Both the items should get selected**SUCCESS**
Click one item and shift+click on another item -> The entire range of items gets selected. **SUCCESS**
When multiple items are selected, observe "Add to World" and "Rename" buttons are disabled **SUCCESS**
Select multiple items and click "Delete" **DONE**
Observe resulting modal displays the count of items marked for deletion. **SUCCESS**
Observe the selected items are deleted **SUCCESS**

Note: When a single item is selected, the Delete modal displays if the item being deleted is a file/folder and its path.  **SUCCESS**
When multiple items are selected, it just displays the number of items marked for deletion.  **SUCCESS**

**Pull request merged**